includes/miscellaneous.inc

```diff
@@ -49,7 +49,19 @@ function bee_initialize_console() {
     chdir($_bee_backdrop_root);
     $_bee_backdrop_root = getcwd();
     define('BACKDROP_ROOT', $_bee_backdrop_root);
-    $_SERVER['HTTP_HOST'] = !empty($_bee_backdrop_site) ? $_bee_backdrop_site : basename($_bee_backdrop_root);
+
+    if (isset($_bee_options['base-url'])) {
+      $url_parts = parse_url($_bee_options['base-url']);
+      $_SERVER['HTTP_HOST'] = $url_parts['host'] . (isset($url_parts['port']) ? ':' . $url_parts['port'] : '');
+      $_SERVER['HTTPS'] = $url_parts['scheme'] === 'https';
+      $GLOBALS['base_url'] = $_bee_options['base-url'];
```

Comment:
What's this used for? Seems unnecessary and confusing...

```suggestion
```

Reply:
I've added a comment to explain that.

Reply:
Ah, I see. This is how the whole thing works basically. `settings.php` can set this, but this is what Nate was trying to avoid. So setting this global variable is essentially setting the value that `settings.php` defines.